pages/Real-world usage.md

```diff
@@ -82,3 +89,9 @@ defmodule MyApp.Redix do
   end
 end
 ```
+
+And then to use the new wrapper in your appplication:
+```elixir
+MyApp.Redix.command(~w(PING))
+```
+Note: You no longer need to pass `conn` as the wrapper is picking one at random.
```

Comment:
I would remove this "Note:", as I think the purpose of having `MyApp.Redix` is clear enough; the example above makes it even clearer.